content/docs/gke/gcp-e2e.md

```diff
@@ -532,19 +532,20 @@ Next, upload the container image to Container Registry so that you can run it on
     kustomize edit add configmap mnist-map-training   --from-literal=learningRate=0.01
     ```
 
-1. Configure parameters and save the model to Cloud Storage:
+1. Configure parameters for where the training results and exported model will be saved in Cloud Storage.  We'll use a timestamp-based subdirectory so that if the tutorial is run more than once, the training will start fresh each time.
 
     ```
-    kustomize edit add configmap mnist-map-training   --from-literal=modelDir=gs://${BUCKET_NAME}/
-    kustomize edit add configmap mnist-map-training   --from-literal=exportDir=gs://${BUCKET_NAME}/export
+    export BUCKET_PATH=${BUCKET_NAME}/$(date +%s)
```

Comment:
It might make more sense to use VERSION_TAG here instead of grabbing a new timestamp, so the folder name matches the container tag that generated the output

Reply:
That's a good idea.